manual/manual/library/Makefile

```diff
@@ -90,7 +95,7 @@ $(INTF): $(MLIS)
 	mv -f Ast_mapper.tex Astmapper.tex
 
 Tk.tex: tk.mli
-	$(CSLDIR)/byterun/ocamlrun $(CSLDIR)/ocamldoc/ocamldoc -latex \
+	$(OCAMLDOC) -latex \
```

Comment:
It was clearly more natural to change this part than leave it out of sync, and I think I don't need to waste time thinking about it; if you think it is dead, you could remove it as a separate commit.